requirements.txt

```diff
@@ -10,16 +10,43 @@
 # `Pipfile.lock` and then regenerate `requirements*.txt`.
 ################################################################################
 
-coverage
-coveralls
-dj-database-url
-django
-django-cors-headers
-django-extensions
-django-heroku
-djangorestframework
-gunicorn
-jwt
-pipenv-to-requirements
-psycopg2-binary
-python-decouple
+asn1crypto==0.24.0
+certifi==2019.3.9
+cffi==1.12.2
+chardet==3.0.4
+coreapi==2.3.3
```

Comment:
@dorothyas Why is your `pipenv_to_requirements generating all these dependencies?